src/c-api.adoc

```diff
@@ -814,9 +814,114 @@ statements, including both RISC-V specific and common operand modifiers.
 [id=function-multi-version]
 == Function Multi-version
 
-Function multi-versioning(FMV) provides an approach to selecting the appropriate
-function according to the runtime environment. The final binary may contain all
-versions of the function, with the compiler generating all supported versions
-and the runtime selecting the appropriate one.
+Function multi-versioning (FMV) allows selecting the appropriate function based on the runtime environment. The binary may contain multiple versions of the function, with the compiler generating all supported versions and selecting the appropriate one at runtime.
 
-This feature is triggered by `target_version/target_clones` function attribute.
+This feature is activated by the `target_version/target_clones` function attributes.
+
+=== Extension Bitmask
+
+The Extension Bitmask is used to probe whether features are enabled during runtime. This is achieved through three bitmask structures: `__riscv_feature_bits` for standard extensions,  `__riscv_vendor_feature_bits` for vendor-specific extensions and `__riscv_cpu_model` for the CPU model. Additionally, `__init_riscv_feature_bits` is used to update the contents of these structures according to the system configuration.
+
+The bitmask structures use the following definitions:
+
+```c
+struct {
+    unsigned length;
+    unsigned long long features[];
+} __riscv_feature_bits;
+
+struct {
+    unsigned length;
+    unsigned long long features[];
+} __riscv_vendor_feature_bits;
+
+struct {
+  unsigned mvendorid;
+  unsigned marchid;
+  unsigned mimpid;
```

Comment:
I wonder why they are not `long` since `marchid` and `mimpid` are XLEN-sized CSR, and Linux will return u64 in hwprobe, which takes the value from `long` typed sbi ecall.

Reply:
Resolved via #91